lerna.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
looks like you've got rid of lerna in favour of the amazing pnpm workspace implementation which is super nice